src/scripts/api.js

```
@@ -1,394 +1,341 @@
+                },
+                ...options
+            });
+            if (res.ok) {
+                let json = await res.json();
+                console.info(`APIRQ OK after ${new Date() - calledAt} ms (@ ${this.identifier})`);
+                resolve(json);
+            } else {
+                console.info(`APIRQ ERR: ${res.status}.`);
+                reject(res);
+            }
+        });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)